platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginCheckInService.java

```diff
@@ -47,47 +53,29 @@ public DefaultGradleEnterprisePluginCheckInService(
     }
 
     // Used just for testing
+    @VisibleForTesting
     public static final String UNSUPPORTED_TOGGLE = "org.gradle.internal.unsupported-enterprise-plugin";
+    @VisibleForTesting
     public static final String UNSUPPORTED_TOGGLE_MESSAGE = "Enterprise plugin unsupported due to secret toggle";
```

Comment:
🤔 Is there any value in introducing a new `org.gradle.internal.unsupported-develocity-plugin` flag that prints out some message about Develocity? Since this is only used for testing on both sides (DV and BT), I seems like it is not worth it.

Actually, we could rename both of those things directly and then change testing on the DV side. WDYT?

Reply:
Is it used for cross-version testing on the DV side? If not, then it's okay to rename, I guess.

Reply:
Yes, it is used in cross version testing, though we already select a different name for the flag depending on the Gradle version: https://github.com/gradle/dv/blob/8f5c79c4800520aa5ba657c723c783e1286a75c6/build-agent-gradle-scans-test-func/src/testFixtures/groovy/com/gradle/scan/plugin/test/func/BaseGradlePluginFuncTest.groovy#L486-L490

So I'd say let's rename it.